spec/system/compare_spec.rb

```diff
@@ -390,6 +401,65 @@
     it_behaves_like "an index page", tab: 'Choose country', show_your_group_tab: false
   end
 
+  context "Admin user" do
+    let(:user) { create(:admin) }
+
+    context "'Country' filter tab" do
+      before { click_on 'Choose country' }
+
+      it_behaves_like "an index page", tab: 'Choose country', show_your_group_tab: false
+      it { expect(page).to have_content "Compare schools by country" }
+      it { expect(page).to have_content "Limit to funder (admin only option)"}
+
+      it_behaves_like "a form filter", id: '#country', country: "All countries"
+      it_behaves_like "a form filter", id: '#funder', funder: "All schools"
+
+      context "Benchmark page" do
+        include_context 'benchmarks page context'
+        before do
+          within '#country' do
+            choose 'Scotland'
+            uncheck 'Middle'
+          end
+          within '#funder' do
+            select 'Funded by Grant Funder'
+          end
+          within '#country' do
+            click_on 'Compare schools'
+          end
+        end
+
+        it_behaves_like "a benchmark list page"
+        it_behaves_like "a filter summary", country: "Scotland", school_types_excluding: ['middle']
+        it_behaves_like "a filter summary", funder: "Grant Funder"
+
+        context "Changing options" do
+          before { click_on "Change options" }
+
+          it_behaves_like "an index page", tab: 'Choose country', show_your_group_tab: false
+          it_behaves_like "a form filter", id: '#country', country: 'scotland', school_types_excluding: ['middle']
+          it_behaves_like "a form filter", id: '#country', funder: "Funded by Grant Funder"
```

Comment:
Should the id for this be '#funder'?

Reply:
It's checking within the countries tab (`#country`) for the expected state of the funder select box, so this is correct (and passes).

Reply:
Sorry yes I can see that now 👍 Was a bit of a swift review, apologies